Marlin/Marlin_main.cpp

```diff
@@ -5956,7 +5962,8 @@ inline void gcode_M17() {
           nozzle_timed_out |= thermalManager.is_heater_idle(e);
 
       #if ENABLED(ULTIPANEL)
-        if (nozzle_timed_out) ensure_safe_temperature();
+        if (nozzle_timed_out)
+          lcd_advanced_pause_show_message(ADVANCED_PAUSE_MESSAGE_CLICK_TO_HEAT_NOZZLE);
```

Comment:
I actually want to re-work this section. Currently, after it unloads, it instructs the user to load the new filament. If the heater idle timeout it expires and tells the user to press the button to re-throw. If the user presses the button, it re-heats, but immediately loads the filament when it gets up to temp. This could be confusing if the user wasn't at the printer when the load filament message was displayed, but they get there when the press to re-heat message is being displayed. I went back and I believe this is how M600 worked before #6407, so I don't think I changed anything at this point.